src/zenml/enums.py

```diff
@@ -323,3 +323,11 @@ class ResponseUpdateStrategy(StrEnum):
     ALLOW = "allow"
     IGNORE = "ignore"
     DENY = "deny"
+
+
+class MetadataResourceTypes(StrEnum):
+    """All possible resource types for adding metadata."""
+
+    PIPELINE_RUN = "pipeline run"
+    STEP_RUN = "step run"
```

Comment:
I'd suggest underscore naming convention instead of spaces:

```suggestion
    PIPELINE_RUN = "pipeline_run"
    STEP_RUN = "step_run"
```

Reply:
[b48e13f](https://github.com/zenml-io/zenml/pull/2064/commits/b48e13fc098f5f27cf68a5789079e6a9da69872a)